.github/workflows/deploy-daac.yml

```diff
@@ -23,9 +23,9 @@ jobs:
             deploy_ref: refs/heads/main
             job_files: job_spec/AUTORIFT.yml job_spec/INSAR_GAMMA.yml job_spec/RTC_GAMMA.yml
             instance_types: r6id.xlarge,r5dn.xlarge,r5d.xlarge
-            default_max_vcpus: 1000
-            expanded_max_vcpus: 2000
-            required_surplus: 2000
+            default_max_vcpus: 1500
+            expanded_max_vcpus: 3000
+            required_surplus: 3000
```

Comment:
Historically we've kept the `hyp3-edc-uat` environment configured the same as `hyp3-edc-prod` so our testing environment is identical. I'm leaning towards updating the uat values to match, although I'm open to setting quota/vcpus/surplus/budget to values more appropriate to test rather than matching prod. Thoughts?

Reply:
I like keeping them exactly the same as well. I suppose we could technically double-spend our budget that way...